src/three-components/ModelScene.js

```diff
@@ -76,21 +78,28 @@ export default class ModelScene extends Scene {
     this.model = new Model();
     this.shadow = new StaticShadow();
     this.light = new AmbientLight(0xffffff, 1);
+    this.light.name = 'AmbientLight';
```

Comment:
Was `name` removed from objects at some point?

Reply:
@cdata `name` is default an empty string, so this helps some debugging